### ISSUE TITLE
1.0.7b1 Recorded notes (cyan light) in CV record mode output something else

### ISSUE THREAD

**Issue by onoma2** [polyend/PresetIssues#1]
Hello,

Just tried new beta-firmware.
Notes recorded with usual (REC+press knob+Wiggle/press pads) process, outputs correct notes while recording, once recording is stopped, the output emits some cyclical low voltage pattern related to recording.

Didn't have time to check if it works with unquantized cvs and gates

**Comment by knobknows** [comment]
yes, I also experienced the same with 1.07. It works fine with v1.06 though.
